app/build.gradle

```diff
@@ -57,12 +63,12 @@ dependencies {
   //automation tests
   androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
   androidTestCompile 'com.android.support.test:runner:0.5'
-  androidTestCompile 'com.android.support:support-annotations:25.0.1'
+  androidTestCompile "com.android.support:support-annotations:${supportLibraryVersion}"
 
   //networking
-  compile 'com.squareup.retrofit2:retrofit:2.1.0'
-  compile 'com.squareup.retrofit2:converter-gson:2.0.0'
-  compile 'com.squareup.okhttp3:logging-interceptor:3.5.0'
+  compile "com.squareup.retrofit2:retrofit:${retrofitVersion}"
+  compile "com.squareup.retrofit2:converter-gson:${retrofitVersion}"
+  compile "com.squareup.okhttp3:logging-interceptor:${okHttpVersion}"
```

Comment:
Consider specifying `compile "com.squareup.okhttp3:okhttp:${rootProject.okhttpVersion}"`. OkHttp gets updated more frequently than Retrofit, so this is currently bringing in an old version of OkHttp.